cleanlab/models/keras.py

```diff
@@ -110,7 +119,7 @@ def fit(self, X, y=None, **kwargs):
             assert_valid_inputs(X, y)
             kwargs["y"] = y
 
-        self.net.fit(X, **kwargs)
+        self.net.fit(X, **self.params, **kwargs)
```

Comment:
Here you might raise a `SyntaxError: keyword argument repeated:`.

Should any repeated keyword argument override the parameters during the call to `fit`?

Reply:
In my head the design choice is that keyword arguments (if passed) could be allowed to override params previously stored, although I do understand the potential confusion. Perhaps I can throw a warning to let users know some params have been changed, or would just a straight up error be better to prevent users from rewriting their own params?

@jwmueller wondering if you have any insight/opinions

Reply:
You have to either update `self.params` with the relevant keys from `kwargs`, before the method call.
```suggestion
        self.net.fit(X, **self.params)
```
or merge these dictionaries during unpacking:
```suggestion
        self.net.fit(X, **{**self.params, **kwargs})
```

In both cases, keys in `self.params` would be overridden by those in `kwargs`.

Reply:
I think allowing them to override the params previously stored seems fine to me.
I'd go with Elias' second suggestion, but I don't have much preference, so whatever you think is best @huiwengoh !